boards/riscv/esp32c3_devkitm/esp32c3_devkitm.dts

```diff
@@ -12,7 +12,18 @@
 	model = "esp32c3_devkitm";
 	compatible = "espressif,esp32c3";
 
+	aliases {
+		uart-0 = &uart0;
```

Comment:
If you use `DT_INST_FOREACH_STATUS_OKAY` in the driver, then you can remove this alias.

Reply:
updated, thank you!